lib/mix/tasks/scenic.gen.component.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Mix.Tasks.Scenic.Gen.Component do
+  @shortdoc "Creates scaffolding for a new Scenic component."
+
+  @moduledoc """
+  Generates Scene boilerplate for a new component.
+
+  ```bash
+  mix scenic.gen.component component_name
```

Comment:
Oh, I completely forgot about documentation!
```suggestion
  mix scenic.gen.component ComponentName
```